model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -158,6 +158,9 @@ class Cluster {
 	// List of machine pools on this cluster.
 	link MachinePools []MachinePool
 
+    // List of node pools on this cluster
+    link NodePools []NodePool
```

Comment:
out of curiosity what is a node pool? and how is it different to machine pools?

Reply:
It's a similar concept but for Hosted clusters.
These NodePools are "detached" from the control plane.
From OCM, the original work created different endpoints for them.

But from ROSA CLI we may invoke one or another depending on hosted cluster or not, that would be my next task and I'm enabling this in the model as a prior step (it wasn't updated before).

Reply:
okay makes sense, can we add that to the comments, as the comments are used to generate the docs in the openapi. Be good to call out the difference between node pools and machine pools.